aten/src/ATen/native/native_functions.yaml

```diff
@@ -1973,9 +1973,9 @@
     NestedTensorCPU, NestedTensorCUDA: NestedTensor_embedding
   autogen: embedding.out
 
-- func: embedding_backward(Tensor grad, Tensor indices, int num_weights, int padding_idx, bool scale_grad_by_freq, bool sparse) -> Tensor
+- func: embedding_backward(Tensor grad, Tensor indices, SymInt num_weights, int padding_idx, bool scale_grad_by_freq, bool sparse) -> Tensor
```

Comment:
hum, I surprised this worked. I would have guessed you'd need

```
dispatch:
  CompositeImplicitAutograd: embedding_backward_symint
```

Reply:
hmm, i think it worked? but i could have not tested it enough yet